src/components/ItaliaTheme/Blocks/ArgumentsInEvidence/Block.jsx

```diff
@@ -32,10 +36,26 @@ const Block = ({
   intl,
 }) => {
   const argument = data?.argument ? data?.argument[0] : null;
+  const searchResults = useSelector((state) => state.content?.subrequests);
```

Comment:
invece di fare una chiamata per ogni argomento, che potrebbe risultare molto oneroso, si può provare a vedere di restituire sempre l'informazione dell'icona tutte le volte che un oggetto di tipo argomento viene richiesto a Plone, cosi si evitano 1000 chiamate. Su questo si può sentire da @cekk che mi pare abbia fatto qualcosa di simile in visitmodena

Reply:
C'è un problema nel comportamento dei sottoblocchi, cioè quando andiamo a salvare un sottoblocco le informazioni vengono salvate all'interno del blocco. Se quindi andiamo a modificare qualcosa (in questo caso un argomento titolo, descrizione, icona) la modifica non viene fatta nel sottoblocco salvato che avrà ancora le informazioni precedenti. L'unico modo per avere le informazioni aggiornate è fare una chiamata asincrona, sbaglio? se non mi sono spiegato magari ne parliamo a voce

Reply:
ahh giusto, non ci avevo pensato a questo caso specifico.. ok allora direi che va bene cosi!